controller/xt_hs_random/impl.go

```diff
@@ -0,0 +1,41 @@
+package xt_hs_random
+
+import (
+	"github.com/netfoundry/ziti-fabric/controller/xt"
+	"math/rand"
+)
+
+func NewFactory() xt.Factory {
+	return factory{}
+}
+
+type factory struct{}
+
+func (f factory) GetStrategyName() string {
+	return "hs-random"
```

Comment:
Maybe we could simplify `hs-random` to `random` and `xt_hs_random` to just `xt_random`?

Reply:
:+1: